services/wallet/token/downloader/main.go

```diff
@@ -100,18 +132,37 @@ func main() {
 		return
 	}
 
+	for _, t := range tokens {
+		t.TokenListID = key
+	}
+
+	version, timestamp := bytesToVersionAndTime(body)
+	token.TokensSources[key] = token.TokensSource{
```

Comment:
hmm editing a global variable from an offline-running script raises a couple flags in my mind.
- When running this offline we won't have access to these fields, unless there's some magic I'm not aware of
- Whenever we decide to run this in runtime you'll want to make access to that struct thread-safe since the token list is accessed from a bunch of different places and this will probably get updated in response to some time-based event.

Reply:
That's not what I wanted, opened the PR a bit early.